lib/web/integrations_awsoidc.go

```diff
@@ -300,6 +304,7 @@ func (h *Handler) awsOIDCConfigureDeployServiceIAM(w http.ResponseWriter, r *htt
 		fmt.Sprintf("--aws-region=%s", shsprintf.EscapeDefaultContext(awsRegion)),
 		fmt.Sprintf("--role=%s", shsprintf.EscapeDefaultContext(role)),
 		fmt.Sprintf("--task-role=%s", shsprintf.EscapeDefaultContext(taskRole)),
+		fmt.Sprintf("--aws-account=%s", shsprintf.EscapeDefaultContext(parsed.AccountID)),
```

Comment:
If some bad actor guesses the integration name, they will get access to the account id as well.
AccountID is not considered a secret, but being able to get it from a public/no-auth endpoint might raise some security concerns.

Reply:
addressed by passing the account id as a query param